docker/ccf_app_js.virtual

```diff
@@ -7,9 +7,8 @@ FROM mcr.microsoft.com/ccf/app/run-js:2.0.7-sgx
 # Note: libjs_generic.virtual is not included in run-js container
 COPY --from=builder /opt/ccf/lib/libjs_generic.virtual.so /usr/lib/ccf
 
-COPY --from=builder /opt/ccf/bin/*.js /app/
-
-# copy config files and member0 certificates into image
+# copy configuration into image
+COPY ./constitution/*.js /app/
```

Comment:
Could we read the constitution files from the run temp directory "./workspace/constitution" like certs and configs files to keep the Docker file generic and not related to each app's folder structure?

Reply:
I think going forward, each sample should have its own config, dockerfile and constitution. It could be that a particular sample needs a particular version of CCF or a different constitution. I think by keeping these things generic we are actually performing scripting gymnastics and means the docker files already are confusing as they get copied to another folder before they are built.

As an example if you look in the dockerfiles we have references to `./workspace` paths, but that folder exists in the banking-app - so it's not intuitive for people trying to work out what is going on.

I also see that we could have an IP address actually specified in the cchost_config file and each sample could have its own IP address so you could spin up the samples at the same time in docker.

In summary, I would actually like to move the dockerfile and config down a directory.